upup/pkg/fi/cloudup/bootstrapchannelbuilder/bootstrapchannelbuilder.go

```diff
@@ -712,7 +712,7 @@ func (b *BootstrapChannelBuilder) buildAddons(c *fi.ModelBuilderContext) (*Addon
 		key := "storage-gce.addons.k8s.io"
 
 		{
-			id := "v1.7.0"
+			id := "k8s-1.22"
```

Comment:
Per our naming convention, this suggests the manifest applies for k8s 1.22 and above. But the code is using it unconditionally, meaning it applies for all supported k8s versions, meaning 1.18 and above. If you intend to cherry-pick this to kOps 1.22, then it would apply to k8s 1.17 and above.

```suggestion
			id := "k8s-1.17"
```